xarray/backends/zarr.py

```diff
@@ -178,19 +178,40 @@ def _determine_zarr_chunks(enc_chunks, var_chunks, ndim, name, safe_chunks):
     raise AssertionError("We should never get here. Function logic must be wrong.")
 
 
+def _get_nczarr_dims(zarr_obj):
+    # NCZarr defines dimensions through metadata in .zarray
+    zarray_path = os.path.join(zarr_obj.path, ".zarray")
+    zarray = zarr.util.json_loads(zarr_obj.store[zarray_path])
```

Comment:
I'm not sure `zarr.util` is a public API. Maybe better to use the `json` module directly here?

Reply:
Good point, I'm now explicitly using `json.loads`